src/commands/misc/copy.md

```diff
@@ -25,27 +25,3 @@ by later copy runs.
 the entire snapshot(s) due to the different encryption keys used in the source
 and destination repository. This *may incur higher bandwidth usage and costs*
 than expected during normal backup runs.
-
-**Important**: The copying process does not re-chunk files, which may break
-deduplication between the files copied and files already stored in the
-destination repository. This means that copied files, which existed in both the
-source and destination repository, *may occupy up to twice their space* in the
-destination repository. See below for how to avoid this.
```

Comment:
Is this being different now, that we leave this out?

Reply:
This is copied from restic docu and doesn't apply to rustic. rustic checks if the chunker parameters are identical, so this cannot occur.